### PR TITLE
Small code enhancements

### PR DESCRIPTION
**Pull request by kirugan**

### CODE REVIEW

File: core/contract.go [hunk 1]
```diff
@@ -9,7 +9,7 @@ import (
 	"github.com/NethermindEth/juno/db"
 )
 
-const contractStorageTrieHeight = 251
+const ContractStorageTrieHeight = 251
```
Comment (pnowosie) [inline core/contract.go:12]: Just curious from where this number comes?
Is it set by us, kind-of big enough value, or it has to be exactly `251`.
Consider adding a comment here.

**Review by pnowosie — APPROVED** [review]
Nice 👍

File: core/contract.go (hunk 1 above)
Comment (kirugan) [reply core/contract.go:12]: I believe it's always fixed (see https://docs.starknet.io/documentation/architecture_and_concepts/Network_Architecture/starknet-state/#:~:text=storage_root%20is%20the%20root%20of%20another%20Merkle%2DPatricia%20trie%20of%20height%20251%20that%20is%20constructed%20from%20the%20contract%E2%80%99s%20storage.)

Comment (kirugan) [reply core/contract.go:13]: added

**Pull request merged**